docs/devguide.md

```diff
@@ -297,6 +298,36 @@ If you choose third party resources storage, the helm chart will not launch an
 etcd server, but will instead instruct the API server to store all resources in
 the Kubernetes cluster as third party resources.
 
+## Dependency Management
+We use [dep](https://golang.github.io/dep) manage our dependencies. We commit the resulting
+vendor directory to ensure repeatable builds and isolation from upstream source disruptions.
+Because vendor is committed, you do not need to interact with dep unless you are
+changing dependencies.
+
+* Gopkg.toml - the dep manifest, this is intended to be hand-edited and contains a set of
+constraints and other rules for dep to apply when selecting appropriate versions of dependencies.
+* Gopkg.lock - the dep lockfile (generated). For the most part, don't edit this file because dep
+completely overwrites that file during `dep ensure`. If you run into trouble with this file, ask.
+* vendor/ - the source of all of our dependencies. Commit changes to this directory in a 
+separate commit from any other changes (including to the Gopkg files) so that it's easier to 
+review your pull request.
+
+If you use VS Code, we recommend installing the [dep extension](https://marketplace.visualstudio.com/items?itemName=carolynvs.dep).
+It provides snippets and improved highlighting that makes it easier to work with dep.
```

Comment:
I'll have to look for an emacs plugin...

Reply:
I actually use Goland, not VS Code, and really wish someone would make a plugin for that for me! 😀 I wimped out and made that dep extension for VS Code instead only because it's so darn easy to write plugins for VS code.

Reply:
I think @n3wscott mentioned that a few weeks or months ago. Maybe he has a tip?